pkg/api/customization/project/project_actions.go

```diff
@@ -26,6 +28,8 @@ import (
 func Formatter(apiContext *types.APIContext, resource *types.RawResource) {
 	resource.AddAction(apiContext, "setpodsecuritypolicytemplate")
 	resource.AddAction(apiContext, "exportYaml")
+	resource.AddAction(apiContext, "enableMonitoring")
```

Comment:
logic should be smarter such that `enableMonitoring` is only showed when montioring is currently disabled and `disbleMonitoring` is only shown when its currently enabled. The UI will decide to hide or show these options based off of the presence or absence of these actions